ui.frontend/src/main/webpack/site/js/components/loadMoreButton.ts

```diff
@@ -73,6 +67,11 @@ const buildLoadMoreButton = ({
     }
   })
 
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  loadMoreButton.addEventListener('click', () => {
+    loadMoreButton.dispatchEvent(onLoadMoreClick)
```

Comment:
Same here with event.target.dispatchEvent :D you know the drill

Reply:
This is done